bcgov/HOUS-permit-portal
# Feature/bphh 363/initial test (validate PDFs)

## Description

This sets up the inital test to validate pdfs.  The last spec will pass once we configure a proper test signed file.  Asking for a test file rather than a file with real data (which this branch was tested with).

## What type of PR is this? (check all applicable)

- [x ] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-363

## Steps to QA

Run rspec and make sure they all pass.  This hits the sandbox api for validating signatures.
